### ISSUE TITLE
Reader - Full Post page & Recent dataviews - sometimes Focus loss on load

### ISSUE THREAD

**Issue by Addison-Stavlo** [Automattic/wp-calypso#99724]
Sometimes when loading a full post page (noticed through recent dataviews mode), the focus gets thrown out of the browser window.

To Repro
* Keyboard nav to select different dataview site streams, then use more keyboard nav or j/k shortkeys to walk through posts in the stream.
* You will notice that some posts load and throw the focus out of the window. pressing `tab` or keyboard shortcuts will no longer have their effect and may throw an error sound until you click back into the window.

Im not entirely sure what causes this. I noticed it yesterday some when doing something else and it seemed consistent with loading site feeds in dataviews, today it is less consistent but im seeing it happen to posts with either:
* blog post embeds
* Videos

This may have something to do with embeds loading? I will continue to try and keep an eye out for this for more info.

**Comment by davemart-in** [comment]
I tested this and couldn't seem to replicate it. If you can record a video of it happening, that might be helpful.

**Comment by Addison-Stavlo** [comment]
I just tested it again a minute ago and it was all good until i got to a post with a blog post embed.

admittedly, i have these kinds of focus loss issues while working on my mac in general. I will also try to repro on my windows machine, and will try to get a video if i can consistently repro.

**Comment by jartes** [comment]
📌 REPRODUCTION RESULTS
- Tested on Reader – Could Not Replicate / Uncertain

📌 ACTIONS
- Triaged → In Triage
- Requested author feedback

📌 Message to Author
@Addison-Stavlo I tried to replicate it, but I'm not sure which keys I should use to "select different dataview site streams". Can you expand on that? Many thanks!

**Comment by Addison-Stavlo** [comment]
>  I will also try to repro on my windows machine, and will try to get a video if i can consistently repro.

I could not repro on my windows machine, in the same posts i had issues with side-by-side on my mac.

After updating the chrome version on my mac, I can no longer repro the issue. I also tried edge and firefox and cannot repro the issue now.

Since no one else can repro either, il close this out and open it if i see it again.
